pom.xml

```diff
@@ -27,7 +27,7 @@
         <prosessering.version>1.20220624132237_7f5ba9c</prosessering.version>
         <changelist>-SNAPSHOT</changelist>
         <start-class>no.nav.familie.ef.iverksett.ApplicationKt</start-class>
-        <familie.kontrakter.version>2.0_20220712141639_0440c10</familie.kontrakter.version>
+        <familie.kontrakter.version>2.0_20220803112940_a98594f</familie.kontrakter.version>
```

Comment:
Denne er litt gammel. Koden din er basert på Periode i stedet for Månedsperiode.

Reply:
Oppdaterte nå.